snooty/rstspec.toml

```diff
@@ -1191,6 +1191,14 @@ type = {link = "http://mongodb.github.io/node-mongodb-native/3.6/%s"}
 help = """Link to a page in the Node.js driver's API reference."""
 type = {link = "http://mongodb.github.io/node-mongodb-native/3.6/api/%s"}
 
+[role.node-api-4.0]
+help = """Link to a page in Node.js driver V4.0's API reference."""
+type = {link = "https://mongodb.github.io/node-mongodb-native/4.0%s"}
+
+[role.node-api-3.6]
+help = """Link to a page in Node.js driver V3.6's API reference."""
+type = {link = "http://mongodb.github.io/node-mongodb-native/3.6/api/%s"}
```

Comment:
It seems like the link for this role is the same as the existing [node-api](https://github.com/mongodb/snooty-parser/pull/317/files#diff-8c18bfff17bea8d40e6b0338264e6176ec2b78618336920fa42b71c5409dd88fR1192)'s link. Is there a reason for wanting to keep both roles?

Reply:
Yes, we plan to update all existing `node-api` directives to either `node-api-3.6` or `node-api-4.0`. This is outlined in these tickets:

[DOCSP-16379](https://jira.mongodb.org/browse/DOCSP-16370)
[DOCSP-15981](https://jira.mongodb.org/browse/DOCSP-15981)

Once all of the `node-api` directives have been migrated to one of the two versions, we hope to delete the `node-api` directive.

Let me know if there is any additional background that could be helpful.

Reply:
Thanks for the context! That sounds reasonable to me. Just make sure it's okay with other teams (if they use the `node-api` role in their content) before it's time to delete the role from the parser to minimize any errors.